src/main/java/frc/trigon/robot/subsystems/collector/placeholdercollector/PlaceholderCollectorConstants.java

```diff
@@ -0,0 +1,102 @@
+package frc.trigon.robot.subsystems.collector.placeholdercollector;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.*;
+import frc.trigon.robot.constants.RobotConstants;
+
+public class PlaceholderCollectorConstants {
+    static final boolean FOC_ENABLED = true;
+    private static final int
+            COLLECTING_MOTOR_ID = 1,
+            ANGLE_MOTOR_ID = 2,
+            ENCODER_ID = 1;
+    private static final InvertedValue
+            COLLECTING_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive,
+            ANGLE_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue
+            COLLECTING_MOTOR_NEUTRAL_MODE = NeutralModeValue.Coast,
+            ANGLE_MOTOR_NEUTRAL_MODE = NeutralModeValue.Coast;
+    private static final AbsoluteSensorRangeValue ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final SensorDirectionValue ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double ENCODER_OFFSET = 0;
+    private static final FeedbackSensorSourceValue ENCODER_TYPE = FeedbackSensorSourceValue.FusedCANcoder;
+    private static final double
+            MOTION_MAGIC_VELOCITY = 80,
+            MOTION_MAGIC_ACCELERATION = 160,
+            MOTION_MAGIC_JERK = 1600;
+    private static final double
+            ANGLE_P = 0.1,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ANGLE_KA = 0,
+            ANGLE_KG = 0,
+            ANGLE_KS = 0,
+            ANGLE_KV = 0;
+    private static final GravityTypeValue ANGLE_GRAVITY_TYPE = GravityTypeValue.Arm_Cosine;
```

Comment:
No need for this as a constant. This is pretty straightforward - this is an arm mechanism, therefore it'll always have an arm cosine gravity type. This is similar to not having a constant for enabling vcomp